debian/clearwater-etcd.init.d

```diff
@@ -208,7 +209,7 @@ do_start()
                      --data-dir $DATA_DIR/$advertisement_ip
                      --name $ETCD_NAME"
 
-        start-stop-daemon --start --quiet --background --make-pidfile --pidfile $PIDFILE --exec $DAEMON --chuid $NAME -- $DAEMON_ARGS $CLUSTER_ARGS \
+        start-stop-daemon --start --quiet --background --make-pidfile --pidfile $PIDFILE --exec $DAEMONWRAPPER --chuid $NAME -- $DAEMON_ARGS $CLUSTER_ARGS \
```

Comment:
You've tested so maybe I'm wrong about this, but I've heard that you should use `--startas` instead of `--exec` if the process name won't be correct at runtime (as in this case).